Client/Frontend/Browser/Browser.swift

```diff
@@ -163,16 +163,20 @@ class Browser: NSObject {
     }
 
     var displayURL: NSURL? {
-        if let url = webView?.URL ?? lastRequest?.URL {
-            if !AboutUtils.isAboutHomeURL(url) {
-                if ReaderModeUtils.isReaderModeURL(url) {
-                    return ReaderModeUtils.decodeURL(url)
-                }
+        if let url = webView?.URL ?? lastRequest?.URL where !AboutUtils.isAboutHomeURL(url) {
```

Comment:
You can drop the `where` condition since there's nothing special about `about:home`.